_sass/_disclaimer.scss

```diff
@@ -1,16 +1,12 @@
 .disclaimer {
   width: 100%;
-  background-color: $color-bright;
+  height: 80%;
```

Comment:
`height: 80%` is a bit of an odd decision - why are you defining this in % rather than in `em`s?